src/main/scala/io/scalac/tezos/translator/actor/EmailSender.scala

```diff
@@ -24,13 +24,14 @@ class EmailSender(service: Emails2SendService,
     case SendEmails => sendEmails
   }
 
-  protected def sendEmails: Future[Unit] = {
-    service
-      .getEmails2Send(batchSize)
-      .map(_.foreach(sendSingleMail))
-  }
+  protected def sendEmails: Future[Unit] =
```

Comment:
what it gives you?

Reply:
canceptually - it gives me that if any of the futures srated by `sendSingleMail` fail, the `sendEmails` will fail.
In this case it doesnt change anything since there are recoveries done under the hood, but IMO its a bad practice to call a `Future` within `foreach`

Reply:
I was rather asking if the previous way (not for comprehension) was not good here (just visual view of the processing source code)